src/ontology/uberon-edit.obo

```diff
@@ -39921,7 +39916,6 @@ intersection_of: part_of UBERON:0002186 ! bronchiole
 relationship: contributes_to_morphology_of UBERON:0002186 ! bronchiole
 relationship: has_part CL:0002145 {source="FMA"} ! ciliated columnar cell of tracheobronchial tree
 relationship: part_of UBERON:0005039 ! mucosa of bronchiole
-property_value: external_ontology_notes "in FMA2 and FMA3, lobular bronchiole is an exact synonym of bronchiole; however there are two distinct terms for the corresponding epithelium." xsd:string {external_ontology="FMA"}
```

Comment:
Since there is still a dbxref from FMA, consider keeping this note or updating it to reflect that historically Uberon considered bronchiole and 'lobular bronchiole' as exact synonyms.

Reply:
It is strange that FMA has a separate class for "lobular bronchiole" and at the same time is a synonym of "bronchiole". The comment has been added again (line 44033).